src/VirtoCommerce.ExperienceApiModule.Core/Extensions/ResolveFieldContextExtensions.cs

```diff
@@ -34,7 +35,14 @@ public static T GetValue<T>(this IResolveFieldContext resolveContext, string key
         public static T GetValue<T>(this IResolveFieldContext resolveContext, string key)
         {
             return resolveContext.GetValue(key, default(T));
-        }     
+        }
+
+        public static string GetCurrentUserId(this IResolveFieldContext resolveContext)
+        {
+            return ((GraphQLUserContext)resolveContext.UserContext).User?.FindFirstValue(ClaimTypes.NameIdentifier)
```

Comment:
Can you move `((GraphQLUserContext)resolveContext.UserContext).User` to variable?

Reply:
Yes, I can!